yb-voyager/cmd/exportData.go

```diff
@@ -541,6 +540,18 @@ func renameDbzmExportedDataFiles() error {
 		if err != nil {
 			return fmt.Errorf("failed to rename dbzm exported data file: %w", err)
 		}
+
+		//rename table schema file as well
```

Comment:
I kept it similar to datafiles, as it will either rename it or change the table name while reading the file for the case-sensitive/reserved words. So the logic to rename all case-sensitive/reserved word table files was already here, I applied the same to schema files as well.